Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP.py

```diff
@@ -0,0 +1,3560 @@
+                md5=dict_safe_get(data_list[0], ['file', 'identity', 'md5']),
+                sha1=dict_safe_get(data_list[0], ['file', 'identity', 'sha1']),
+                sha256=file_hash,
+                path=dict_safe_get(data_list[0], ['file', 'file_path']),
+                name=dict_safe_get(data_list[0], ['file', 'file_name']),
+                hostname=dict_safe_get(data_list[0], ['computer', 'hostname']),
+                dbot_score=dbot_score
+            )
+
+            for data in data_list[1:]:
+                disposition = dict_safe_get(data, ['file', 'disposition'])
+                dbot_score = get_dbotscore(client.reliability, file_hash, disposition)
+
+                file_indicator.md5 = file_indicator.md5 or dict_safe_get(data, ['file', 'identity', 'md5'])
+                file_indicator.sha1 = file_indicator.sha1 or dict_safe_get(data, ['file', 'identity', 'sha1'])
+                file_indicator.path = file_indicator.path or dict_safe_get(data, ['file', 'file_path'])
+                file_indicator.name = file_indicator.name or dict_safe_get(data, ['file', 'file_name'])
+                file_indicator.hostname = file_indicator.hostname or dict_safe_get(data, ['computer', 'hostname'])
+                file_indicator.dbot_score = file_indicator.dbot_score or dbot_score
+
+                is_all_filled = file_indicator.md5 \
+                    and file_indicator.sha1 \
+                    and file_indicator.sha256 \
+                    and file_indicator.path \
+                    and file_indicator.name \
+                    and file_indicator.hostname \
+                    and file_indicator.dbot_score
+
+                if is_all_filled:
+                    break
+
+            file_context = file_indicator.to_context().get(Common.File.CONTEXT_PATH)
+            readable_output = tableToMarkdown(
+                f'Cisco AMP - Hash Reputation for: {file_hash}',
+                file_context
+            )
+
+            command_results.append(CommandResults(
+                readable_output=readable_output,
+                outputs_prefix='',
+                raw_response=raw_response,
+                outputs_key_field='SHA256',
+                indicator=file_indicator
+            ))
+
+        except Exception as exc:  # pylint: disable=broad-except
+            command_results.append(CommandResults(
+                readable_output=f'Could not process file: "{file_hash}"\n{str(exc)}')
+            )
+
+    return command_results
+
+
+''' HELPER FUNCTIONS '''  # pylint: disable=pointless-string-statement
+
+
+def get_pagination_parameters(
+    page: int = None,
+    page_size: int = None,
+    limit: int = None
+) -> Pagination:
+    """
+    Get the limit and offset required for the http request,
+    number of requests required and if the pagination is automatic, manual or none of them.
+
+    Args:
+        page (int, optional): Page number to view. Defaults to None.
```

Comment:
why is the default `None`? Why not 0 or 1?

Reply:
Fixed to 0.